ACCS-534: My Account: Delete a stored card in My Account (#820)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/user-account/events.mdx

| Array of token rows | Each item matches the **StoredPaymentMethodDisplay** shape: `publicHash`, `cardBrand`, `lastFourDigits`, optional `expired`, optional `variant`. |
| `null` | Explicit empty list (same convention as cart `cart/data`). |

> When the list comes from the event bus, the shopper still confirms in the removal modal first; after confirmation, the row is removed from **local state only** and the **deletePaymentToken** mutation is **not** called.

Reviewer: ```suggestion
When the list comes from the event bus, the shopper still completes the remove confirmation dialog first. After that, the row is removed from **local state only** and the **deletePaymentToken** mutation is **not** called.
```
